<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-2G86F66K" name="com.CloudBlaze.Onboarding.Subscription" tests="13" failures="1" timestamp="26 Feb 2020 08:36:14 GMT" time="48.958" errors="0">
  <testcase name="verifySubscriptionPage" time="3.525" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyUrlOfSubscription" time="0.158" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifySubscriptionPageComponents" time="0.336" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyGridComponents" time="0.072" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyColumnsOfGrid" time="0.194" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyNumberOfRecords" time="0.014" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyOnClickOfCancelAllFieldShouldGetClear" time="10.029" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyOnClickOfSubmitButtonAlertIsComing" time="6.020" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyDeleteDialogBox" time="2.161" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifySubscriptionIsCreated" time="10.637" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifySameNameSubscription" time="3.709" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyDeleteRecord" time="4.206" classname="com.CloudBlaze.Onboarding.Subscription"/>
  <testcase name="verifyEditButton" time="7.897" classname="com.CloudBlaze.Onboarding.Subscription">
    <failure type="java.lang.AssertionError" message="Updation is not succesfully expected [Subscription updated successfully!] but found []">
      <![CDATA[java.lang.AssertionError: Updation is not succesfully expected [Subscription updated successfully!] but found []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.CloudBlaze.Onboarding.Subscription.verifyEditButton(Subscription.java:512)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyEditButton -->
</testsuite> <!-- com.CloudBlaze.Onboarding.Subscription -->
